krizten/Memoire-WebClient
# #165524661 Develop Signup Page UI

### What does this PR do?
- Build out the user interface for the signup page/
#### Description of Task to be completed?
- Convert the UI/UX design provided to markup.
- Ensure the signup page is responsive to screen sizes and matches the design provided.
#### How should this be manually tested?
- Clone this repository and checkout to the branch: ```ft/165524661/signup-page```
- Ensure [yarn](https://yarnpkg.com/lang/en/) is installed, then install the project dependencies by simply running ```yarn``` in the terminal.
- Start the project using ```yarn start```.
- Using a web browser, navigate to ```http://localhost:3000/signup``` to see the signup page.
#### What are the relevant pivotal tracker stories?
#165524661
#### Screenshots (if appropriate)
![memoire-signup-page-desktop](https://user-images.githubusercontent.com/29978455/57200521-fe0ad600-6f84-11e9-8d84-00256d4f748c.png)
![memoire-signup-page-mobile](https://user-images.githubusercontent.com/29978455/57200523-04994d80-6f85-11e9-916c-eb3d2507f16a.png)
